notebooks/SITCOMTN_101_FCU_investigation.ipynb

```diff
@@ -0,0 +1,1216 @@
+{
```

Comment:
<p>I did a first quick scan in the notebook and my first comment is associated with the format. I recommend you use empty lines to group together pieces of coherent code. Leaving some space between these "intrinsic logic blocks of code" help reading and understanding it better. </p><p> </p><p>I will run it in my computer now to have a better evaluation. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/sitcomtn-101/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>